src/services/widgetTemplate/publish.ts

```diff
@@ -16,12 +16,14 @@ interface CreateWidgetTemplateReq {
     channel_id: number;
 }
 
+const channelId = process.env.WIDGET_BUILDER_CHANNEL_ID ? parseInt(process.env.WIDGET_BUILDER_CHANNEL_ID, 10) : 1;
```

Comment:
Do we initially as for channelId information during the config step?

Reply:
We dont, its defaulted to `WIDGET_BUILDER_CHANNEL_ID=1` after `init`